test/analysis/typeComplexityAnalysis.test.ts

```diff
@@ -1008,17 +1009,48 @@ describe('Test getQueryTypeComplexity function', () => {
                     hero(episode: EMPIRE) { 
                         id, name 
                     } 
-                    human(id: 1) @includes(if: $directive) { 
+                    human(id: 1) @skip(if: $directive) { 
```

Comment:
The weight of Human and hero are both 1. Can you add something that differentiates the weights so we know that this is working.